documentation/modules/security/proc-renewing-your-own-ca-certificates.adoc

```diff
@@ -16,92 +17,68 @@ The procedure describes the renewal of CA certificates in PEM format.
 
 * The Cluster Operator is running.
 * xref:installing-your-own-ca-certificates-{context}[Your own CA certificates and private keys are installed].
-* You have new cluster and clients X.509 certificates and keys in PEM format.
-
-These could be generated using an `openssl` command, such as:
-
-[source,shell,subs="+quotes"]
-openssl req -x509 -new -days _NUMBER-OF-DAYS-VALID_ --nodes -out ca.crt -keyout ca.key
+* You have new cluster and clients X.509 certificates in PEM format.
 
 .Procedure
 
-. Check the details of the current CA certificates in the `Secret`:
-+
-[source,shell,subs="+quotes"]
-kubectl describe secret _CA-CERTIFICATE-SECRET_
-+
-_CA-CERTIFICATE-SECRET_ is the name of the `Secret`, which is `_KAFKA-CLUSTER-NAME_-cluster-ca-cert` for the cluster CA certificate and `_KAFKA-CLUSTER-NAME_-clients-ca-cert` for the clients CA certificate.
+Before going through the following steps, make sure that the CN (Common Name) of the new CA certificate is different from the current one.
+For example, when the Cluster Operator renews certificates automatically it adds a _v<version_number>_ suffix to identify a version.
+Do the same with your own CA certificate by adding a different suffix on each renewal.
 
-. Create a directory to contain the existing CA certificates in the secret.
+. Update the corresponding `Secret` with the new CA certificate and generation.
 +
-[source,shell,subs="+quotes"]
-----
-mkdir new-ca-cert-secret
-cd new-ca-cert-secret
-----
-
-. Fetch the secret for each CA certificate you wish to renew:
+.. Edit the existing `Secret`:
 +
 [source,shell,subs="+quotes"]
-kubectl get secret _CA-CERTIFICATE-SECRET_ -o 'jsonpath={.data._CA-CERTIFICATE_}' | base64 -d > _CA-CERTIFICATE_
+kubectl edit secret _CA-CERTIFICATE-SECRET_
 +
-Replace _CA-CERTIFICATE_ with the name of each CA certificate.
-
-. Rename the old `ca.crt` file as `ca-__DATE__.crt`,
-where _DATE_ is the certificate expiry date in the format _YEAR-MONTH-DAYTHOUR-MINUTE-SECONDZ_.
+_CA-CERTIFICATE-SECRET_ is the name of the `Secret`, which is `_KAFKA-CLUSTER-NAME_-cluster-ca-cert` for the cluster CA certificate and `_KAFKA-CLUSTER-NAME_-clients-ca-cert` for the clients CA certificate.
 +
-For example `ca-2018-09-27T17-32-00Z.crt`.
+Following an example of the `Secret` related to the cluster CA certificate for a Kafka cluster named `my-cluster`.
 +
-[source,shell,subs="+quotes"]
-mv ca.crt ca-$(date -u -d"$(openssl x509 -enddate -noout -in ca.crt | sed 's/.*=//')" +'%Y-%m-%dT%H-%M-%SZ').crt
+[source,yaml,subs=attributes+]
+----
+apiVersion: v1
+kind: Secret
+data:
+  ca.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0F... <1>
+metadata:
+  annotations:
+    strimzi.io/ca-cert-generation: "0" <2>
+  labels:
+    strimzi.io/cluster: my-cluster
+    strimzi.io/kind: Kafka
+  name: my-cluster-cluster-ca-cert
+  #...
+type: Opaque
+----
+<1> Current CA certificate base64 encoded
+<2> Current CA certificate generation
```

Comment:
Seems a bit weord to have the before example and no after example => should we have both before and after (shows the bumped generation and new CRT file)?

Reply:
Ok. I will add the after example in the section related to "Save changes and exit".